### PR TITLE
Update phone number code - part 1

### PR DESCRIPTION
**Pull request by gcapalbo**
Updating messaging code to reference the new postgres PhoneNumber model instead of the old couch VerifiedNumber model.

@calellowitz

### CODE REVIEW

File: corehq/apps/sms/tests/test_phone_numbers.py [hunk 1]
```diff
@@ -223,3 +229,216 @@ def test_filter_pending(self):
...
+        number1.save()
+        number2.phone_number = '999224'
+        number2.save()
+        self.assertIsNone(PhoneNumber.by_suffix('1 23'))
+        self.assertIsNone(PhoneNumber.by_suffix('2 23'))
+        self.assertEqual(PhoneNumber.by_suffix('124'), number1)
+        self.assertEqual(PhoneNumber.by_suffix('224'), number2)
+
+        # test pending
+        number1.verified=False
```
Comment (dimagimon) [inline corehq/apps/sms/tests/test_phone_numbers.py]: E225 missing whitespace around operator

File: corehq/apps/sms/tests/test_phone_numbers.py [hunk 2]
```diff
@@ -223,3 +229,216 @@ def test_filter_pending(self):
...
+        number1.save()
+        number2.verified=False
```
Comment (dimagimon) [inline corehq/apps/sms/tests/test_phone_numbers.py]: E225 missing whitespace around operator

File: corehq/apps/sms/tests/test_phone_numbers.py [hunk 3]
```diff
@@ -223,3 +229,216 @@ def test_filter_pending(self):
...
+        self.assertIsNone(PhoneNumber.by_phone('999124'))
+
+        # test cache clear on save
+        number.phone_number = '999124'
+        number.save()
+        self.assertIsNone(PhoneNumber.by_phone('999123'))
+        self.assertEqual(PhoneNumber.by_phone('999124'), number)
+
+        # test pending
+        number.verified=False
```
Comment (dimagimon) [inline corehq/apps/sms/tests/test_phone_numbers.py]: E225 missing whitespace around operator

**Comment by calellowitz** [comment]
looks good to me. will wait for tests but ready to merge

**Comment by gcapalbo** [comment]
(don't merge yet, build failed and didn't run on the latest commit)

**Comment by gcapalbo** [comment]
ok, good to go now :white_check_mark:

**Pull request merged**